benchmark/src/test/resources/ai/timefold/solver/benchmark/impl/result/resultValues.txt

```diff
@@ -0,0 +1 @@
00/-186765.00,994906.00/-186760.00,995481.00/-186513.00,996561.00/-186459.00,997387.00/-186322.00,998911.00/-186238.00,999342.00/-186164.00,1000197.00/-186075.00,1014551.00/-186051.00,1015368.00/-185786.00,1016573.00/-185700.00,1017752.00/-185431.00,1018119.00/-185368.00,1020555.00/-185276.00,1021059.00/-185199.00,1021924.00/-185178.00,1022704.00/-185099.00,1023452.00/-184918.00,1023766.00/-184791.00,1025521.00/-184675.00,1026062.00/-184517.00,1027215.00/-184424.00,1028463.00/-184310.00,1029122.00/-184242.00,1095809.00/-184209.00,1097494.00/-184047.00,1097893.00/-184011.00,1100470.00/-183950.00,1100836.00/-183858.00,1102317.00/-183776.00,1102995.00/-183634.00,1104455.00/-183593.00,1104663.00/-183537.00,1105256.00/-183488.00,1105822.00/-183312.00,1106472.00/-183248.00,1107239.00/-183057.00,1164959.00/-183005.00,1165585.00/-182686.00,1168667.00/-182580.00,1209749.00/-182403.00,1210404.00/-182193.00,1210630.00/-182105.00,1213936.00/-182034.00,1214493.00/-181889.00,1215534.00/-181775.00,1216248.00/-181532.00,1216447.00/-181383.00,1221144.00/-181340.00,1221897.00/-181104.00,1222589.00/-181013.00,1225922.00/-180953.00,1226369.00/-180889.00,1227384.00/-180612.00,1227700.00/-180534.00,1272221.00/-180499.00,1273385.00/-180368.00,1854419.00/-180270.00,1855112.00/-180138.00,1855497.00/-179917.00,1856308.00/-179734.00,1858102.00/-179716.00,1858741.00/-179660.00,1859651.00/-179414.00,1859734.00/-179409.00,1860364.00/-179322.00,1862928.00/-179288.00,1863888.00/-179155.00,1864927.00/-179077.00,1865504.00/-178962.00,1866850.00/-178800.00,1875767.00/-178732.00,1876119.00/-178549.00,1876500.00/-178364.00,1876859.00/-178326.00,1877477.00/-178107.00,1878371.00/-178016.00,1880603.00/-177909.00,1881055.00/-177839.00,1981547.00/-177752.00,1982084.00/-177685.00,1983260.00/-177538.00,1983719.00/-177368.00,1984698.00/-177307.00,1985432.00/-177240.00,1985684.00/-177176.00,1985932.00/-177097.00,1986561.00/-177075.00,1987304.00/-176939.00,1989111.00/-176869.00,1990041.00/-176703.00,1990258.00/-176613.00,1991035.00/-176487.00,1992117.00/-176389.00,2523485.00/-176360.00,2597077.00/-176217.00,2597595.00/-176096.00,2600931.00/-176005.00,2601430.00/-175817.00,2604282.00/-175755.00,2604596.00/-175664.00,2604935.00/-175576.00,2605731.00/-175450.00,2606843.00/-175412.00,2607241.00/-175379.00,2607902.00/-175290.00,2610423.00/-175145.00,2868654.00/-174996.00,2868848.00/-174878.00,2869944.00/-174636.00,2871747.00/-174568.00,2872171.00/-174510.00,2873069.00/-174246.00,2873352.00/-174097.00,2874196.00/-174067.00,2875034.00/-174014.00,2875714.00/-173917.00,2876790.00/-173857.00,2878191.00/-173748.00,2878412.00/-173669.00,2883184.00/-173600.00,2885176.00/-173568.00,2885601.00/-173501.00,2886618.00/-173390.00,2887661.00/-173351.00,2888256.00/-173279.00,2889078.00/-173135.00,2889566.00/-172886.00,2889891.00/-172796.00,2890498.00/-172623.00,2896148.00/-172536.00,2896411.00/-172515.00,2896817.00/-172423.00,2897498.00/-172287.00,2897693.00/-172191.00,3589914.00/-172070.00,3600000.00/-172046.00,
```

Comment:
Ideally this file would have the same format as the input files.
And all those input files would be called not `.txt`, but `.csv`, because that is what they are - comma-separated values.